core/templates/pages/moderator-page/moderator-page.mainpage.html

```diff
@@ -18,7 +18,7 @@
             </ul>
           </navbar-breadcrumb>
           <content>
-            <oppia-moderator-page></oppia-moderator-page>
+            <oppia-moderator-page class="protractor-test-moderator-page"></oppia-moderator-page>
```

Comment:
[optional] Should we validate a user-facing button/heading/message to ensure the presence of a moderator page? (I generally see the e2e test as an end-user testing the feature)

Reply:
In this case we're really checking permissions--we're checking that users cannot access the moderator page while moderators can. Since we're not checking the functionality of the moderator page (yet), I think it's OK to not check an element visible to the user. We should add e2e tests for moderator functionality eventually though